packages/opentelemetry-api/src/trace/SamplingResult.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Attributes } from './attributes';
+
+export enum SamplingDecision {
```

Comment:
What is the point of making this a bitmask? Can I arbitrarily combine bits? e.g. is `0b10` a valid sampling decision? Think a standard enum is likely fine in this case.

Reply:
Please add tsdocs to this.